mne/decoding/ssd.py

```diff
@@ -331,6 +330,11 @@ def apply(self, X):
         X = pick_patterns @ X_ssd
         return X
 
+    @property
+    def picks_(self):
+        """Backward compatible wrapper for picks."""
+        return self.picks
```

Comment:
do we want to deprecate `.picks_` or just keep it around forever?

Reply:
It's pretty trivial to keep it I think so probably not worth deprecating

Reply:
see comment above